js/main.js

```diff
@@ -0,0 +1,26 @@
+const $photoInput = document.querySelector('.photo-input');
+function updateValue() {
+  document.getElementById('img').setAttribute('src', $photoInput.value);
```

Comment:
You should be querying for DOM elements outside of functions. When you include it inside a function, you're forcing us to traverse the DOM over and over every single time this function gets called. If you just do it once globally, we can do the search once, and always have a reference to it